app/jobs/appropriate_bodies/process_batch/record_outcome_job.rb

```diff
@@ -0,0 +1,33 @@
+module AppropriateBodies
+  module ProcessBatch
+    # @see AppropriateBodies::ProcessBatch::Action
+    class RecordOutcomeJob < ApplicationJob
+      queue_as :process_batch
+
+      def perform(pending_induction_submission_id, author_email, author_name)
+        pending_induction_submission = PendingInductionSubmission.find(pending_induction_submission_id)
+        pending_induction_submission_batch = pending_induction_submission.pending_induction_submission_batch
+        appropriate_body = pending_induction_submission_batch.appropriate_body
+
+        author = Events::AppropriateBodyBatchAuthor.new(
+          email: author_email,
+          name: author_name,
+          appropriate_body_id: appropriate_body.id,
+          batch_id: pending_induction_submission_batch.id
+        )
+
+        teacher = Teacher.find_by(trn: pending_induction_submission.trn)
```

Comment:
Do we want to raise an error here if the teacher record is missing (i.e `find_by!`), or will the `RecordOutcome` blow up without a teacher anyway?

Reply:
It will blow but I want to do some follow up work and make `RecordOutcome` and `ReleaseECT` behave the same (whilst also doing the model association. They should look very similar IMO but both services are written slightly different.